content/es/docs/tasks/tools/install-kubectl.md

```diff
@@ -116,57 +230,62 @@ Si estás en Windows y usando el gestor de paquetes [Powershell Gallery](https:/
 
 2. Para asegurar que la versión utilizada sea la más actual puedes probar:
 
-    ```
-    kubectl version
+    ```powershell
+    kubectl version --client
     ```
 
-    {{< note >}}Actualizar la instalación se realiza mediante la re-ejecución de los dos comandos listados en el paso 1.{{< /note >}}
+{{< note >}}
+Actualizar la instalación se realiza mediante la re-ejecución de los dos comandos listados en el paso 1.{{< /note >}}
 
-## Instalar en Windows usando Chocolatey o scoop
+### Instalar en Windows usando Chocolatey o scoop
 
-Para instalar kubectl en Windows puedes usar bien el gestor de paquetes [Chocolatey](https://chocolatey.org) o el instalador de línea de comandos [scoop](https://scoop.sh).
-{{< tabs name="kubectl_win_install" >}}
-{{% tab name="choco" %}}
+1. Para instalar kubectl en Windows puedes usar el gestor de paquetes [Chocolatey](https://chocolatey.org) o el instalador de línea de comandos [scoop](https://scoop.sh).
 
+    {{< tabs name="kubectl_win_install" >}}
+    {{% tab name="choco" %}}
+    ```powershell
```

Comment:
```suggestion
Using  [Chocolatey](https://chocolatey.org).

    ```powershell
```

Let's try if adding a text line fixes the rendering issue:

<img width="883" alt="Screenshot 2020-12-16 at 15 08 09" src="https://user-images.githubusercontent.com/795289/102358986-839f9600-3fb0-11eb-88a3-a89b49b3f881.png">

https://deploy-preview-25481--kubernetes-io-master-staging.netlify.app/es/docs/tasks/tools/install-kubectl/#instalar-en-windows-usando-chocolatey-o-scoop

Reply:
Yeah, weird! Thank you for catching this.